test/external/imports/ts-cjs/tsconfig.json

```diff
@@ -4,12 +4,11 @@
 	"compilerOptions": {
 		"target": "esnext",
 		"module": "commonjs",
-		"declaration": true,
+		"declaration": false,
 		"sourceMap": false,
-		"outDir": "build/",
 		"strict": true,
-		"moduleResolution": "node",
 		"esModuleInterop": true,
-		"skipLibCheck": true
+		"moduleResolution": "node",
```

Comment:
this is the default also, right?

Reply:
`classic` is the default - https://www.typescriptlang.org/tsconfig#moduleResolution